Makefile

```diff
@@ -74,12 +70,6 @@ DEFINES := CTEST_OUTPUT_ON_FAILURE=1 \
 
 
 .setup:
-ifneq ($(MAKECMDGOALS),deps)
-ifeq ($(GIT_EXISTS),)
-	@echo "Problem: cannot find 'git'"
```

Comment:
Why remove this check, seems really helpful when debugging :)

Reply:
a.) Either they already have git because they cloned osquery :)
b.) Or if they are using /vagrant path or used a osquery tgz `make sysprep` will install it for them

If we want to keep this check here, we should remove `package git` from all the provision scripts
I removed this check and kept the `package git` in provision scripts.

Reply:
They could be downloading the .tgz provided by GitHub's release interface. My preference is to keep this check as a fail-safe.

Reply:
Ok. I will add it back. I guess that makes `package git` redundant then? Because `make sysprep` will fail with `Problem: cannot find 'git'` before it can install it